### ISSUE TITLE
Accessibility issues with keyboard navigation

### ISSUE THREAD

**Issue by lauradanze** [shahabyazdi/react-multi-date-picker#342]
Greetings and thank you for your work. 
The datepicker is currently not accessible, meaning not usable with keyboard navigation. There are two main issues:

1) Keyboard trap
It is impossible to get out of datepicker, because tab navigation is cycling through the elements (which is fine) but there is no exit key (like ESC) defined. Once a user is entering the datepicker, there is no way to exit it with a keyboard.

2) First week not reachable via keyboard in "Multi Month Mode"
If "number of months" is set to 2 or 3, it is impossible to reach the first week of the second or third month with the keyboard in case those weeks contain days of the previous month. The keyboard navigation is simply jumping over the first week respectively. Those weeks are only reachable, if the entire week is filled with selectable days, like e.g. 01.12.2024 = Sun

Please fix those issues when you find the time. Thank you very much!

Best regards
Laura

**Comment by githubmor** [comment]
is there any solution ?
